<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgOption</class>
 <widget class="QDialog" name="DlgOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放器设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="layoutMain">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frmTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 26, 27);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="lblTitle">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>播放器设置</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-image:url(:/resource/close.png);color: rgb(255, 255, 255);}
QPushButton:hover{border-image:url(:/resource/close_hover.png);color: rgb(255, 255, 255);}
QPushButton:pressed{border-image:url(:/resource/close.png);color: rgb(255, 255, 255);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frmContent">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(26, 26, 27);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabOptions">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #1A1A1B;border:0px;
border-color: rgb(255, 170, 255);</string>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabPlayer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <attribute name="title">
            <string>通用设置</string>
           </attribute>
           <widget class="QPushButton" name="btnSaveTabPlayer">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>220</y>
              <width>151</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btnSaveTabPlayer{border:0px;border-radius:10px;color:#E6E6E6; background-color:#474A51;}
QPushButton#btnSaveTabPlayer:hover{border:0px;border-radius:10px;color:#E6E6E6; background-color:#00B1EA;}
QPushButton#btnSaveTabPlayer:pressed{border:0px;border-radius:10px;color:#E6E6E6; background-color:#007CA3;}
QPushButton#btnSaveTabPlayer:disabled{border:0px;border-radius:10px;color:#7C7C7C; background-color:#35373C;}</string>
            </property>
            <property name="text">
             <string>确定修改</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>2</width>
              <height>270</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(71, 74, 81);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QCheckBox" name="chkHardAccel">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>181</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox{background-color: rgba(0, 0, 0, 0);color:#F0F0F0}
QCheckBox:hover{ background-color: rgba(0, 0, 0, 0);color:#1DB1EA;}</string>
            </property>
            <property name="text">
             <string>硬件加速渲染(重启生效)</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tabExtent">
           <attribute name="title">
            <string>文件格式</string>
           </attribute>
           <widget class="QLabel" name="lblFormatExt">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>10</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{background-color: rgba(0, 0, 0, 0);color:#F0F0F0}</string>
            </property>
            <property name="text">
             <string>支持格式:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>2</width>
              <height>270</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(71, 74, 81);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QTextEdit" name="editFormatExt">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>321</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);color:#F0F0F0</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="lblFormatExt_2">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>90</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{background-color: rgba(0, 0, 0, 0);color:#F0F0F0}</string>
            </property>
            <property name="text">
             <string>用户格式:(例如: .mp3;.mp4)</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="leUserFormat">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>120</y>
              <width>321</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(60, 60, 60, 255);color:#F0F0F0;border-width:0px;
border-color: rgba(60, 60, 60, 255);</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnSaveFormat">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>230</y>
              <width>151</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border:0px;border-radius:10px;color:#E6E6E6; background-color:#474A51;}
QPushButton:hover{border:0px;border-radius:10px;color:#E6E6E6; background-color:#00B1EA;}
QPushButton:pressed{border:0px;border-radius:10px;color:#E6E6E6; background-color:#007CA3;}
QPushButton:disabled{border:0px;border-radius:10px;color:#7C7C7C; background-color:#35373C;}</string>
            </property>
            <property name="text">
             <string>确定修改</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tabAbout">
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <attribute name="title">
            <string>关于...</string>
           </attribute>
           <widget class="QLabel" name="lblSourceCode">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>100</y>
              <width>361</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#f0f0f0;&quot;&gt;源码下载: &lt;/span&gt;&lt;a href=&quot;https://github.com/zhxin999/weplayer&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#00aaff;&quot;&gt;Github&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:10pt; color:#f0f0f0;&quot;&gt; UDP控制接口文档: &lt;/span&gt;&lt;a href=&quot;https://github.com/zhxin999/weplayer/wiki/UDP%E6%8E%A5%E5%8F%A3%E8%AF%B4%E6%98%8E%E6%96%87%E6%A1%A3&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#55aaff;&quot;&gt;Wiki&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QLabel" name="lblVersion">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>60</y>
              <width>361</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#f0f0f0;&quot;&gt;软件版本: 2.0.2 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QLabel" name="lblCopyRight">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>190</y>
              <width>291</width>
              <height>25</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 124);</string>
            </property>
            <property name="text">
             <string>开发者: zhxin999@163.com, lqy_china@163.com</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>2</width>
              <height>270</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(71, 74, 81);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="lblCopyRight_2">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>230</y>
              <width>251</width>
              <height>25</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(124, 124, 124);</string>
            </property>
            <property name="text">
             <string>本播放器基于FFMPEG和Qt开发</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>361</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;WePlayer, 干净，简单，可以第三方播放控制视频播放器&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>140</y>
              <width>361</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#f0f0f0;&quot;&gt;联系作者: &lt;/span&gt;&lt;a href=&quot;zhxin999@163.com&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#00aaff;&quot;&gt;zhxin999@163.com&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:10pt; color:#f0f0f0;&quot;&gt;微信号: zhxin999&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
           <zorder>lblSourceCode</zorder>
           <zorder>lblVersion</zorder>
           <zorder>lblCopyRight</zorder>
           <zorder>lblCopyRight_2</zorder>
           <zorder>label_5</zorder>
           <zorder>label_4</zorder>
           <zorder>label_7</zorder>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
